source/Ship.h

```diff
@@ -309,6 +309,12 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double Hull() const;
 	double Fuel() const;
 	double Energy() const;
+	double DisplaySolar() const;
+	double DisplayRamScoop() const;
+	// These are for the thruster activity bars
+	double DisplayThrust() const;
```

Comment:
remove the definition of the function beneath to avoid overloading error

Reply:
Thanks! done.